CONTRIBUTING.md

```diff
@@ -181,3 +212,7 @@ This procedure will be changed to the DRAFT section below.
 ### (DRAFT) Rollback
 
 You can use the same procedure as deploying, just select a different artifact.
+
+```
```

Comment:
What are we trying to do here?

Reply:
Hmm.... I don't know why this is added... I'll remove it